streaming/src/main/scala/org/apache/spark/streaming/Checkpoint.scala

```diff
@@ -41,6 +41,13 @@ class Checkpoint(ssc: StreamingContext, val checkpointTime: Time)
   val checkpointDuration = ssc.checkpointDuration
   val pendingTimes = ssc.scheduler.getPendingTimes().toArray
   val sparkConfPairs = ssc.conf.getAll
+  @transient val recoverableAccuNameToId = ssc.recoverableAccuNameToId
```

Comment:
the exception is about the `ssc` (StreamingContext), probably the property of `recoverableAccuNameToId`; in the meantime, the calculation of `val trackedAccs = ` should happens in master node, not in every executor, I doubt this exception was caused by something else?